config/manifests-crd/bases/recipe.clusterserviceversion.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: recipe.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Recipe is the Schema for the recipes API
+      displayName: Recipe
+      kind: Recipe
+      name: recipes.ramendr.openshift.io
+      version: v1alpha1
+  description: Recipe description. TODO.
+  displayName: Recipe
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
```

Comment:
I was mildly confused at this scope, but as there is no operator we should be fine.

Reply:
This is required as the recipe crd bundle will be installed by the ODF operator in its operator group. The install modes of all the CSVs in the operator group should match. Just copying the installModes of ODF operator here. It has no impact on how the recipe CRD is reconciled as that is decided by the installMode of the recipe controller(CSV).